opengever/maintenance/scripts/set_iworks_mimetypes.py

```diff
@@ -0,0 +1,50 @@
+import mimetypes
+from opengever.maintenance.debughelpers import setup_app
+from opengever.maintenance.debughelpers import setup_option_parser
+from opengever.maintenance.debughelpers import setup_plone
+from os.path import splitext
+from plone import api
+import transaction
+
+
+def set_iworks_mimetype():
+    """Search all MS OneNote documents and fixes the contentType.
+    """
+
+    catalog = api.portal.get_tool('portal_catalog')
+    brains = catalog.unrestrictedSearchResults(
+        portal_type='opengever.document.document')
+    # A getIcon value of 'application.png' indicates that the mimetype
+    # coulnd't be resolved for this document - only consider these
+    brains = filter(lambda b: b.getIcon == 'application.png', brains)
+    for brain in brains:
```

Comment:
this is potentially long-running, could you use a `ProgressLogger`, see https://github.com/4teamwork/opengever.maintenance/blob/81f586b0eb61a3908bb7a28e6e2375c0ba9affd2/opengever/maintenance/scripts/bumblebee_installation.py#L140 or an example.